### ISSUE TITLE
The ‘msvd_inceptivev4.hdf5’ I extracted is different from the one you provided, I want to know what caused it?

### ISSUE THREAD

**Issue by xiaoyongbest1234** [hobincar/RecNet#6]
Hi, why did I extract `msvd_inceptivev4.hdf5` with the size of 10.3GB using the [`pytorch-video-feature extractor`](https://github.com/hobincar/pytorch-video-feature-extractor), while you only provided 634.4MB?
My `MSVD dataset` was downloaded from[`here`](http://www.cs.utexas.edu/users/ml/clamp/videoDescription/YouTubeClips.tar). Is there an exception problem with the `MSVD dataset `I downloaded?Or did some of my actions cause 10.3GB?
In addition, can you provide a link to download the `MSVD/ MSR-VTT` dataset?

I look forward to your reply and thank you very much.

**Comment by hobincar** [comment]
Hi,

I extract the InceptionV4 features for MSVD again by just cloning the repository to make sure I have the same code with you, and I got an hdf5 file of size 606MB. It is still different from what I already uploaded (I extracted the 636MB features a long time ago), but the 10.3GB features you got seems to be not normal.

I downloaded the MSVD dataset from [here](http://www.cs.utexas.edu/users/ml/clamp/videoDescription/#data) (which is the same as yours), and the MSR-VTT dataset from [here](https://github.com/VisionLearningGroup/caption-guided-saliency/issues/6).

To make sure anyone can extract features themselves from the [repo](https://github.com/hobincar/pytorch-video-feature-extractor), I'll upload the newly extracted features of size 606MB after I check its performance.
